modules/commands/index.js

```diff
@@ -122,6 +132,16 @@ module.exports = function (client) {
       });
     }
 
+    // Check for misspelt command and suggest it
+    if (!item) {
+      const matches = stringSimilarity.findBestMatch(trigger, commandNames);
+      const bestMatch = matches?.bestMatch;
+      if (bestMatch.rating >= config.similaritySensitivity ?? 0.5) {
+        await message.channel.send(`Did you mean \`!${bestMatch.target}\`?\nType \`!help\` for a list of commands.`);
+        return;
+      }
+    }
+
```

Comment:
I think this could be merged in with the following `if (!item)` check, the response is slightly similar, you can just append to the string if there is a match.

Reply:
For sure. I had it in there originally, but figured to split it up

Reply:
Changed in aa94dc8